src/ray_tracer.cpp

```diff
@@ -117,24 +117,26 @@ simulateRangeScan (const nm::OccupancyGrid& grid, const gm::Pose& sensor_pose,
   {
     const double theta = scanner_info.angle_min+i*scanner_info.angle_increment;
     const gm::Point scan_max =
-      rayEndPoint(p0, theta0 + theta, scanner_info.range_max+1);
+      rayEndPoint(p0, theta0 + theta, scanner_info.range_max + grid.info.resolution);
```

Comment:
Yeah, It is probably enough with ```range_max```  but I kept an additional safety pixel just in case.